### PR TITLE
fix(docs): disable js minimization in docusaurus to display code samples properly

### PR DESCRIPTION
**Pull request by ShaiBer**
*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Review by staycoolcall911 — APPROVED** [review]

File: docusaurus.config.js
Comment (MarkMcCulloh) [inline docusaurus.config.js:1]: How is this config file used? the actual config is [here](https://github.com/winglang/docsite/blob/main/docusaurus.config.js) no?

**Review by MarkMcCulloh — CHANGES_REQUESTED** [review]

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,14 +8,21 @@ module.exports = {
...
-      ],
-    },
+  configureWebpack: (config, isServer, utils) => {
+    const { getCacheLoader } = utils;
+    return {
+      module: {
+        rules: [
+          {
+            test: /\.(w|js|ts|tf)$/,
+            use: 'file-loader',
```
Comment (MarkMcCulloh) [inline docusaurus.config.js:18]: I believe webpack 5 does not need this anymore, you can use [asset/resource](https://webpack.js.org/guides/asset-modules/)

Comment (MarkMcCulloh) [reply docusaurus.config.js:18]: note this section as well https://webpack.js.org/guides/asset-modules/#replacing-inline-loader-syntax

File: docusaurus.config.js
Comment (ShaiBer) [reply docusaurus.config.js:1]: That's a good question.. I assumed they somehow get merged.
The reason I didn't use the one in the docsite is that I wanted to test these things locally and when I work locally I don't even have the docsite repo.
Is there some other way to solve this?

Comment (MarkMcCulloh) [reply docusaurus.config.js:1]: afaik there is no current process for this file to be used in the build process.

Not 100% sure of the best approach right now, I think the workflow deserves some thought.

Comment (ShaiBer) [reply docusaurus.config.js:1]: Yes.. definitely

**Pull request closed without merging**